docs/cli/chain.md

```diff
@@ -0,0 +1,6 @@
+
+# Chain
+
+The ```chain``` command groups actions to interact with the blockchain in the client:
+
+- [```chain sethead```](./account_new.md): Set the current chain to a certain block.
```

Comment:
Is this related to this PR?

Reply:
Nope, that is from another one I forgot to do before. Actually, the link of that command is incorrect. I will change it.